### DISCUSSION TITLE
Change WorkSpace

### DISCUSSION THREAD

**Discussion by Je-Leo-AS** [Open-Agriculture/AgIsoStack-plus-plus#330] [Q&A] [answered]
Hello, what's up?
I would like to know if this library has any function that changes the workspace of the screen.

**Comment by ad3154** [comment]
Hello there!

As far as I can tell, "workspace" isn't a concept in ISO 11783 by name at least, so I'm inclined to say "no" unless you're referring to something else. For example, if you want to change the active data mask for your own UT/VT application, then yes we would support that with our library, but for example if you wanted to change the active working set on your terminal, there isn't a CAN message defined in ISO11783 to do that since it's the terminal's responsibility to handle that in some way. Likewise, if you want to manipulate a graphics context object on the terminal, we'd support that too as part of a broader application with an appropriate object pool and sufficiently new terminal.

If none of that sounds like what you are looking for, then workspace may be a manufacturer proprietary concept for your specific screen?

**Reply by Je-Leo-AS** [reply]
Thanks for the quick reply!

I see.

By 'workspace' I meant something like the 'application' not the datamask of the application, like for example I created a datamask for a certain application, and I wanted to know if it's possible to change applications through CAN, but from what I understand it won't be possible. Thanks again for your attention!

**Reply by ad3154** [reply, marked as answer]
Ah, okay yeah, those are generally called working sets or sometimes object pools, and unfortunately there is no standard way to change between them via CAN, sorry about that.
